ontohgis.ttl

```diff
@@ -10246,7 +10246,7 @@ ontohgis:object_23 rdf:type owl:Class ;
                                          rdf:type owl:Class
                                        ] ;
                    rdfs:subClassOf ontohgis:object_1 ;
-                   rdfs:isDefinedBy "Budynek jest to przedmiot trwający w czasie, który został wytworzony lub wykorzystany w celu odgraniczenia pewnej przestrzeni (“wewnątrz\") od innej przestrzeni (“zewnątrz”)."@pl ;
+                   rdfs:isdefinedbyLiteral "Budynek jest to przedmiot trwający w czasie, który został wytworzony lub wykorzystany w celu odgraniczenia pewnej przestrzeni (“wewnątrz\") od innej przestrzeni (“zewnątrz”)."@pl ;
```

Comment:
```suggestion
                   rdfs:isDefinedByLiteral "Budynek jest to przedmiot trwający w czasie, który został wytworzony lub wykorzystany w celu odgraniczenia pewnej przestrzeni (“wewnątrz\") od innej przestrzeni (“zewnątrz”)."@pl ;
```

names are case-sensitive. The same applies to all instances of isdefinedbyIRI and isdefinedbyLiteral.

Use the same command as in the previous comment, i.e.:

```bash
sed -i -e 's/\(\s\)\S\+:isDefinedByIRI/\1ontohgis:isDefinedByIRI/gi' \
       -e 's/\(\s\)\S\+:isDefinedByLiteral/\1ontohgis:isDefinedByLiteral/gi' ontohgis.ttl
```

Reply:
Agree. Fixed.